repo/packages/H/hue/0/marathon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "/hue",
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "mesosphere/hue",
+            "privileged": true,
+            "network": "BRIDGE",
+            "portMappings": [
+                {
+                    "containerPort": 8000,
+                    "hostPort": {{hue.port}},
+                    "protocol": "tcp"
+                },
+                {
+                    "containerPort": 14000,
+                    "hostPort": {{httpfs.port}},
+                    "protocol": "tcp"
+                }
+            ]
+        },
+        "volumes": [
+            {
+                "containerPath": "/opt/mesosphere/",
+                "hostPath": "/opt/mesosphere/",
+                "mode": "RO"
+            },{
+                "containerPath": "/etc/hadoop/conf/",
+                "hostPath": "/opt/mesosphere/etc/hadoop/",
+                "mode": "RO"
+            }
+        ]
+    },
+    "uris": [
+        "{{hue.configuration}}",
+        "{{httpfs.configuration}}"
+    ],
+    "cmd": "export PATH=/opt/mesosphere/bin:$PATH && \\cp -f /etc/hadoop/conf/*-site.xml /etc/hadoop-httpfs/conf/ && \\cp -f $MESOS_SANDBOX/httpfs-site.xml /etc/hadoop-httpfs/conf/ && service hadoop-httpfs start; \\cp -f $MESOS_SANDBOX/hue.ini /etc/hue/conf/hue.ini && /usr/lib/hue/build/env/bin/hue runserver 0.0.0.0:8000",
```

Comment:
This is PATH manipulation looks very DCOS Image specific. Is this so we can get access to `service`? The user should be able to overwrite this to work on their env. cc @cmaloney